decidim-elections/spec/commands/decidim/elections/admin/setup_election_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Elections::Admin::SetupElection do
+  subject { described_class.new(form, bulletin_board: bulletin_board) }
+
+  let(:organization) { create :organization, available_locales: [:en, :ca, :es], default_locale: :en }
+  let(:participatory_process) { create :participatory_process, organization: organization }
+  let(:current_component) { create :component, participatory_space: participatory_process, manifest_name: "elections" }
+  let(:user) { create :user, :admin, :confirmed, organization: organization }
+  let(:election) { create :election, :complete }
+  let(:trustees) { create_list :trustee, 5, :considered, :with_public_key }
+  let(:trustee_ids) { trustees.pluck(:id) }
+  let(:errors) { double.as_null_object }
+  let(:form) do
+    double(
+      invalid?: invalid,
+      election: election,
+      current_user: user,
+      current_component: current_component,
+      current_organization: organization,
+      trustee_ids: trustee_ids,
+      errors: errors
+    )
+  end
+  let(:bulletin_board) do
+    Decidim::Elections::BulletinBoardClient.new(
+      server: "http://localhost:8000/api",
+      api_key: "89Ht70GZNcicu8WEyagz_rRae6brbqZAGuBEICYBCii-PTV3MAstAtx1aRVe5H5YfODi-JgYPvyf9ZMH7tOeZ15e3mf9B2Ymgw7eknvBFMRP213YFGo1SPn_C4uLK90G",
```

Comment:
Is this hardcoded in purpose? Maybe it's better to move it to some test helper file?